frb/data/analysis/CIGALE/2MASS_H.dat

```diff
@@ -0,0 +1,175 @@
+13000.0 0.0
```

Comment:
Can you add a header to each of these similar
to what we have for the others?  I guess it is only
for `CIGALE` which we might not use much anymore,
but good for uniformity.

Here is the header for `GMOS_N_r` by way of example:

```
#GMOS_N_r
#energy
#gemini Gmos N r band http://svo2.cab.inta-csic.es/theory/fps/index.php?id=Gemin
i/GMOS-N.r&&mode=browse&gname=Gemini&gname2=GMOS-N#filter
```

Reply:
Added